reconcile/cli.py

```diff
@@ -1184,3 +1185,13 @@ def integrations_validator(ctx):
 def prometheus_rules_tester(ctx, thread_pool_size, cluster_name):
     run_integration(reconcile.prometheus_rules_tester, ctx.obj,
                     thread_pool_size, cluster_name)
+
+
+@integration.command()
+@threaded()
+@binary(['amtool'])
```

Comment:
nice! would this be added to qontract-reconcile-base?

Reply:
Yes, I forgot to create the PR and link it 😅  https://github.com/app-sre/container-images/pull/9